### PR TITLE
Feature/wave spawning

### PR DESCRIPTION
**Pull request by Sherk-cell**
Basic wave spawning.

### CODE REVIEW

File: PVB Linx Groep 6/Assets/Code/WaveSpawner/WaveManager.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class WaveManager : MonoBehaviour
+{
+    [SerializeField] private List<GameObject> enemyPrefabs = new List<GameObject>(0);
+    [SerializeField] private List<EnemySpawner> _spawners = new List<EnemySpawner>(0);
+    public int wave;
+    public int totalZombies;
```
Comment (DaanKikkert) [inline PVB Linx Groep 6/Assets/Code/WaveSpawner/WaveManager.cs]: Change to "totalEnemies"

File: PVB Linx Groep 6/Assets/Code/WaveSpawner/EnemySpawner.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    private Vector2 _zoneSize;
+    
+    
+
+    private void Awake()
+    {
+        _zoneSize.x = zone.localScale.x;
+        _zoneSize.y = zone.localScale.z;
+        var enemyHolderPosX = (_zoneSize.x / 2) * -1;
+        var enemyHolderPosZ = (_zoneSize.y / 2) * -1;
```
Comment (DaanKikkert) [inline PVB Linx Groep 6/Assets/Code/WaveSpawner/EnemySpawner.cs]: Please change from var to specific datatype

**Review by DaanKikkert — CHANGES_REQUESTED** [review]
Read comments

**Comment by Sherk-cell** [comment]
Pull request comments added

**Pull request merged**